import/csv-importer.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+function importMiniJobCards() {
+    echo "\n\n--MINIJOB CARDS--\n\n";
+    $file = file(__DIR__ . "/Minijobs-Table 1.csv");
+    $fileContent = array_slice($file, 2); //removes the first two elements (table name and table header)
+    $tableHeaderItems = array_slice($file, 1, 1); //array element containing the table headers
+    $keys = array_slice(explode(";", $tableHeaderItems[0]), 0, 3); //three table header items
+
+    foreach ($fileContent as $item) {
```

Comment:
Evtl. anderes Naming?
```suggestion
    foreach ($fileContent as $line) {
```